pysac/mhs_atmosphere/parameters/logical.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Dec 14 11:37:39 2014
+
+@author: sm1fg
+
+"""
+##============================================================================
+##logical parameters
+##============================================================================
+def get_logical(model, l_mpi, l_SI=True, l_gdf=True):
+
+    """This module assigns the logical options for the model. If adding 
+    new models with additional logical arguments add it to the default 
+    list as false, include an if statement for True update the
+    dictionary logical_pars 
+    
+    """    
+    #default arguments
+    l_CGS         = False
+    if not l_SI:
+        l_CGS     = True
+    if l_gdf:
+        suffix = '.gdf'
+    else:
+        suffix = ''
+    l_hdonly      = False  # set magnetic field to zero to check background
```

Comment:
Rather than effectively listing these variables twice I would construct the dict here and then edit them in place lower down.